verification/curator-service/ui/package-lock.json

```diff
@@ -41094,6 +40963,20 @@
         "whatwg-fetch": "^3.0.0"
       }
     },
+    "react-beautiful-dnd": {
```

Comment:
Is this part of another PR? Seems to be from the highlighting search terms work.

Reply:
You know what? I don't know. I didn't explicitly put this here, I ran `npm i` to ensure the deps were up to date and I committed the change to the lock file. I can't find `react-beautiful-dnd` in the package.json so I can revert this file if you think it's wrong.

Reply:
Right, so the highlighting work seems to be already in main, which would explain this. I'm fine with keeping this change.
```
verification/curator-service/ui/src/components/ViewCase.tsx
30:import Highlighter from 'react-highlight-words'
```